crates/cast/bin/cmd/send.rs

```diff
@@ -189,20 +195,33 @@ impl SendTxArgs {
         } else {
             // Retrieve the signer, and bail if it can't be constructed.
             let signer = eth.wallet.signer().await?;
+            let zk_signer = eth.wallet.signer().await?;
```

Comment:
https://github.com/matter-labs/foundry-zksync/pull/610/commits/3653a8dec08a5e9de1f8cda50e0959db8304f1c2

Reply:
Need to find another way or drop ledger support